src/Parse/ParseUser.php

```diff
@@ -224,15 +218,30 @@ public static function logInWithTwitter(
                 'Cannot log in Twitter user without an auth token secret.'
             );
         }
+        $authData = [
+            'id'                => $id,
+            'screen_name'       => $screen_name,
+            'consumer_key'      => $consumer_key,
+            'consumer_secret'   => $consumer_secret,
+            'auth_token'        => $auth_token,
+            'auth_token_secret' => $auth_token_secret,
+        ];
+
+        return self::logInWith('twitter', $authData);
+    }
+
+    /**
+     * Logs in with an OAuth service.
+     *
+     * @param string $serviceName the name of OAuth service
```

Comment:
The service here is not necessarily OAuth based, just 'service' should do. A good example would be `ParseUser::loginWithAnonymous`, which should also be updated to use `ParseUser::logInWith`.